nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -225,7 +225,7 @@ public class InvokeHTTP extends AbstractProcessor {
             .build();
 
     public static final PropertyDescriptor SOCKET_CONNECT_TIMEOUT = new PropertyDescriptor.Builder()
-            .name("Connection Timeout")
+            .name("Socket Connection Timeout")
```

Comment:
Why is this rename happening here? I think the original name of 'Connection Timeout' is more concise but also equally clear. But if it were to be renamed, it would to be done so in the migrateProperties method